.eslintrc.json

```diff
@@ -19,8 +19,7 @@
         "no-undef": "warn",
         "no-unused-vars": "warn",
         "no-redeclare": "warn",
-        "no-useless-escape": "warn",
-        "no-mixed-spaces-and-tabs": "warn"
```

Comment:
This feels like a good rule, why remove it?

Reply:
The rule will still be included as it's part of the `eslint:recommended` config. By removing it from here we're simply reverting to its original error level 🙂